client/apps/edit/components/layout/index.coffee

```diff
@@ -73,8 +64,8 @@ module.exports = class EditLayout extends Backbone.View
     }
 
   toggleAstericks: =>
-    @$('.edit-required + :input').each (i, el) =>
-      $(el).prev('.edit-required').attr 'data-hidden', !!$(el).val()
+    @$('.edit-required ~ :input').each (i, el) =>
+      $(el).prevAll('.edit-required').attr 'data-hidden', !!$(el).val()
```

Comment:
Do you see this being a problem at all? The "Use article title" button was added between the `edit-required` and `input` elements which caused this part to break unless I used prevAll. Not sure if there's a better solution or if this affects anything else.